symbology/dictionary-symbol-sample/build.gradle

```diff
@@ -0,0 +1,18 @@
+apply plugin: 'java'
+apply plugin: 'application'
+//apply plugin: 'com.esri.arcgisruntime.java'
+
+buildscript {
+    repositories {
+        maven {
+            url 'https://esri.bintray.com/arcgis'
+        }
+    }
+    dependencies {
+        //classpath 'com.esri.arcgisruntime:gradle-arcgis-java-plugin:0.9.0'
```

Comment:
@tyle8552 
Going to have to undo this later. This sample should work for beta 2 but it's not. I was using build 1380 jar to run this.